css/index.css

```diff
@@ -101,10 +102,10 @@ p {
     justify-content: space-evenly;
 }
 
```

Comment:
You can leave the .box here and use multiple classes in the index.html for each box.  Doing it that way will properly use parent/child relationships and the child will inherit the parent's styles.